src/layouts/components/Header.tsx

```diff
@@ -142,71 +205,96 @@ function DesktopNavbar(): JSX.Element {
-            text="Vaults"
-            pathname="/vaults"
-            testId="Mobile.HeaderLink.Vaults"
-          />
-          <HeaderLink
-            className="flex justify-center border-b border-gray-100 dark:border-gray-700"
-            text="Auctions"
-            pathname="/auctions"
-            testId="Mobile.HeaderLink.Auctions"
-          />
-          <HeaderLink
-            className="flex justify-center border-b border-gray-100 dark:border-gray-700"
-            text="Oracles"
-            pathname="/oracles"
-            testId="Mobile.HeaderLink.Oracles"
-          />
-          <HeaderLink
-            className="flex justify-center border-b border-gray-100 dark:border-gray-700"
-            text="Tokens"
-            pathname="/tokens"
-            testId="Mobile.HeaderLink.Tokens"
-          />
-          <HeaderLink
-            className="flex justify-center border-b border-gray-100 dark:border-gray-700"
-            text="Masternodes"
-            pathname="/masternodes"
-            testId="Mobile.HeaderLink.Masternodes"
+    <div
+      onClick={() => {
+        toggleMenu();
+      }}
+      className="h-screen w-screen backdrop-blur-[2px] backdrop-brightness-50"
+    >
+      <div
+        onClick={(e) => {
+          e.stopPropagation();
+        }}
+        className="flex flex-col float-right h-screen w-5/12 bg-white dark:bg-gray-900 lg:hidden"
+        data-testid="TabletMenu"
+      >
+        <div className="flex flex-row justify-between p-6 items-center">
+          <Link href={{ pathname: "/" }} passHref>
+            <a className="hover:text-primary-500">
+              <DeFiChainLogo className="h-full w-36" />
+            </a>
+          </Link>
+          <MdClose
+            role="button"
+            className="h-6 w-6 text-primary-500"
+            onClick={() => toggleMenu()}
+            data-testid="Header.CloseMenu"
           />
         </div>
-
-        <div className="mt-4" data-testid="Mobile.HeaderSearchBar">
-          <SearchBar atHeader={false} />
+        <div className="flex h-full flex-col ">
+          <div className="flex flex-wrap bg-primary-700 p-4 dark:bg-gray-900">
+            <HeaderCountBar className="flex w-full flex-wrap" />
+            <div className="mt-4 w-full">
+              <HeaderNetworkMenu />
+            </div>
+          </div>
+          <div className="mt-2">
+            <MenuItems viewPort="Tablet" />
+          </div>
         </div>
-      </Container>
+      </div>
+    </div>
+  );
+}
 
-      <div className="p-2 bg-primary-700 dark:bg-gray-900 flex flex-wrap p-4 md:p-0">
-        <HeaderCountBar className="w-full flex flex-wrap" />
-        <div className="w-full mt-4">
+function MobileMenu({ toggleMenu }: { toggleMenu: () => void }): JSX.Element {
+  const ref = useRef<HTMLDivElement>(null);
+  const dimension = useWindowDimensions();
+  useEffect(() => {
+    if (ref.current) {
+      ref.current.style.height = `${dimension.height - 225}px`;
```

Comment:
how do you get `225`?

Reply:
updated to use ref.current.offsetTop to calculate the height